lm_admin_guide.md

```diff
@@ -316,6 +320,57 @@ The only non-Python dependency is **PostgreSQL** (back-end/client), which is
 required by the `psycopg2` Python package.
 
 
+
+## How to upgrade
+
+### Upgrade a Docker deployment
+Existing Dockerized deployments can be easily upgraded to a more recent LifeMonitor version by going through the following steps:
+1. stop `lm` service
+    ```bash
+    docker-compose stop lm
+    ``` 
+2. make a backup of LifeMonitor to a temp path within the `db` container
+    ```bash
+    docker-compose exec db /bin/bash -c "PGPASSWORD=\${POSTGRESQL_PASSWORD} pg_dump -U \${POSTGRESQL_USERNAME} \${POSTGRESQL_DATABASE} > /tmp/lifemonitor_backup.sql"
+    ```
+3. copy backup from the `db` container to a local path
+    ```bash
+    docker cp life_monitor_db_1:/tmp/lifemonitor_backup.sql ${HOME}/lifemonitor_backup.sql
+    ```
```

Comment:
Something seems to be missing here. What's the purpose of `${HOME}/lifemonitor_backup.sql`? Is it needed in order to perform a rollback in case the upgrade fails? If so, we should say so here and then explain how to do the rollback later in the section.

Reply:
Yes, it is needed to perform a rollback (https://github.com/crs4/life_monitor/pull/134/commits/5d7432db37cce86fa2ebf65af2de0c5cfa34fb79)